packages/core/test-helpers/core-test-helpers-so-type-serializer/src/extract_migration_info.ts

```diff
@@ -60,7 +60,7 @@ export const extractMigrationInfo = (soType: SavedObjectsType): SavedObjectTypeM
     const entry = modelVersionMap[version];
     return {
       version,
-      changeTypes: entry.changes.map((change) => change.type),
+      changeTypes: [...new Set(entry.changes.map((change) => change.type))].sort(),
```

Comment:
Why do we want a de-duped, alphabetically sorted list here? Might be missing something, but not sure it provides much value over the the previous listing of changeTypes.

Reply:
sorted: because this structure is exclusively used for type hash generation, and the order of transformations doesn't matter
deduped: more arguable, but because I felt like the purpose of this struct was to list the type of changes that are bound to the model version, not to know the number of each individual type.